test/config/logs.go

```diff
@@ -0,0 +1,51 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/onsi/ginkgo"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// Formatter is the format configuration to write logs into text
+	Formatter = logrus.TextFormatter{
+		DisableTimestamp: true,
+	}
+
+	// TestLogWriter buffer to store all the logs generated by the test
+	TestLogWriter bytes.Buffer
+	// TestLogFileName is the default name to store test logs
+	TestLogFileName = "logs"
+)
+
+// TestLogWriterReset reset the current buffer
+func TestLogWriterReset() {
+	TestLogWriter.Reset()
+}
+
+// LogHook to send logs via `ginkgo.GinkgoWriter`.
+type LogHook struct{}
+
+// Levels defined levels to send logs to FireAction
+func (h *LogHook) Levels() []logrus.Level {
+	return []logrus.Level{
+		logrus.WarnLevel,
+		logrus.ErrorLevel,
+		logrus.FatalLevel,
+		logrus.PanicLevel,
+	}
+}
+
+// Fire is a callback function used by logrus to write logs that match in
+// the given by `Levels` method
+func (h *LogHook) Fire(entry *logrus.Entry) (err error) {
+	line, err := Formatter.Format(entry)
+	if err == nil {
+		fmt.Fprintf(ginkgo.GinkgoWriter, string(line))
+	}
+	fmt.Fprintf(os.Stderr, "LogHook.Fire: unable to format log entry (%v)", err)
```

Comment:
Should this be in an else clause? otherwise we'd print every time? alternatively, we could return inside the `if err == nil` but in that case I'd recommend handling the error and early return in the if and making the default path in the function handle the `err == nil` scenario.